sympy/integrals/prde.py

```diff
@@ -459,24 +459,25 @@ def prde_cancel_liouvillian(b, Q, n, DE):
             ba, bd = frac_in(b, DE.t, field=True)
     elif DE.case == 'exp':
         with DecrementLevel(DE):
-            ba, bd = frac_in(b + n*derivation(DE.t, DE)/DE.t, DE.t)
+            ba, bd = frac_in(b + n*derivation(DE.t, DE)/DE.t, DE.t, field=True)
 
     for N in range(n, -1, -1):
         with DecrementLevel(DE):
             Qyn = [frac_in(q.nth(N), DE.t) for q in Q]
-            f[N], A[N] = param_rischDE(ba, bd, Qyn, DE)
-        f[N] = [Poly(fa.as_expr()/fd.as_expr(), DE.t, field=True)
-                for fa, fd in f[N]]
-
-    r = [v.nullspace() for v in A]
-    # take the first vector of nullspace
-    d = [i[0][m - 1:] for i in r]
-
-    for N in range(n, -1, -1):
```

Comment:
This may not be there in overall files changed list. But seeing this particular commit or other way would be seeing [2nd commit](https://github.com/sympy/sympy/pull/12743/commits/0c8f943d91aada4ac5432056396cbd6d7b99cd12) side-by-side with [3rd commit](https://github.com/sympy/sympy/pull/12743/commits/08a7adc2c266429e3ccc5e5ac3b4d2a5dc3d39a3) of this pull request.